Client/mods/deathmatch/logic/luadefs/CLuaAudioDefs.cpp

```diff
@@ -1389,6 +1393,786 @@ int CLuaAudioDefs::GetSoundEffects(lua_State* luaVM)
     return 1;
 }
 
+int CLuaAudioDefs::SetSoundEffectParameter(lua_State* luaVM)
```

Comment:
Is there any reason to not use the new argument parser for these new functions? I think it'd make for much more readable code, but maybe I'm not realizing some limitation of the new argument parser of something.

Reply:
It would be harder yes, because the second parameter (effect parameter name) depends on the effect.
In this case the old parser provided more readable code IMHO.